interp/dumpglob.ml

```diff
@@ -37,15 +37,15 @@ let dump_to_dotglob () = glob_output := MultFiles
 
 let dump_into_file f =
   if String.equal f "stdout" then
-    (glob_output := StdOut; glob_file := Pervasives.stdout)
+    (glob_output := StdOut; glob_file := Stdlib.stdout)
   else
     (glob_output := File f; open_glob_file f)
 
 let feedback_glob () = glob_output := Feedback
 
 let dump_string s =
   if dump () && !glob_output != Feedback then 
-    Pervasives.output_string !glob_file s
+    Stdlib.output_string !glob_file s
```

Comment:
None of the qualifications in this file seem necessary.

Reply:
I'm OK removing them; I just didn't dare to do it without consensus first.